src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -0,0 +1,136 @@
+/**
+* Components > Cuisines Widget
+* =================================
+*
+* Example: Homepage – Popular cuisines.
+*
+* The `c-cuisinesWidget` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesWidget();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-widget
+*/
+
+//TODO: Improvement of this solution: https://jira.just-eat.net/browse/WH-682 as suggested in https://github.com/justeat/fozzie/pull/188#issuecomment-442138750
```

Comment:
Don't think we should have internal jira links or references on our public github

Reply:
Good point